mm2src/coins/utxo/utxo_common.rs

```diff
@@ -62,14 +63,16 @@ use std::cmp::Ordering;
 use std::collections::hash_map::{Entry, HashMap};
 use std::str::FromStr;
 use std::sync::atomic::Ordering as AtomicOrdering;
+#[cfg(test)]
+use utxo_common_tests::{utxo_coin_fields_for_test, utxo_coin_from_fields};
 use utxo_signer::with_key_pair::{calc_and_sign_sighash, p2sh_spend, signature_hash_to_sign, SIGHASH_ALL,
                                  SIGHASH_SINGLE};
 use utxo_signer::UtxoSignerOps;
 
 pub mod utxo_tx_history_v2_common;
 
 pub const DEFAULT_FEE_VOUT: usize = 0;
-pub const DEFAULT_SWAP_TX_SPEND_SIZE: u64 = 305;
+pub const DEFAULT_SWAP_TX_SPEND_SIZE: u64 = 305; // TODO: check maybe need to change because of the new burn account output
```

Comment:
This needs to be changed in this PR for taker payment spend. I am not even sure if this is accurate for taker funding spend.

Reply:
I guess this needs to change for TPU where a burn output is added to taker payment spend tx.
However, I checked a sample htlc spend tx (from tests so it's komodo like). It's size is 496, with the extra burn output. So w/o the burn output it should be 496-34=462
(for btc, with a smaller header, a similar tx size is 479)
Tried to fix for 496 (this increases tx fee) in [f8b543b](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2112/commits/f8b543b97f2067b830d6d0167af74caad0fe3b33)